django-debug-toolbar/meta.yaml

```diff
@@ -10,6 +10,9 @@ source:
 build:
     number: 1
 
+extra:
```

Comment:
> The skip section is in the build 

Thanks!  The indentation in https://github.com/conda/conda-build/pull/566 made me think it was at the root level.

Anyway, I know it won't do anything for now.  I just left it there as a reminder.  I am actually  hoping @kwilcox fix django dependency this in sci-wms.